src/web/mjs/connectors/LittleGarden.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class LittleGarden extends Connector {
+    constructor() {
+        super();
+        super.id = 'littlegarden';
+        super.label = 'Little Garden';
+        this.tags = ['manga', 'french'];
+        this.url = 'https://littlexgarden.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'h2.super-title');
+        const id = uri.pathname;
+        const title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const request = new Request(new URL('/mangas', this.url), this.requestOptions);
+        const urlList = await this.fetchDOM(request, 'div.listing a.no-select');
+
+        return urlList.map((urlElement) => ({
+            id: this.getRelativeLink(urlElement),
+            title: urlElement.title
+        }));
+    }
+
+    async _getChapters(manga) {
+        const slug = manga.id.substr(1);
+        const operationName = 'chapters';
+        const query = 'query chapters($slug: String, $limit: Float, $skip: Float, $order: Float!, $isAdmin: Boolean!) {\n  chapters(limit: $limit, skip: $skip, where: {deleted: false, published: $isAdmin, manga: {slug: $slug, published: $isAdmin, deleted: false}}, order: [{field: "number", order: $order}]) {\nnumber}\n}\n';
+        const variables = {
+            slug: slug,
+            order: -1,
+            skip: 0,
+            limit: 99999,
+            isAdmin: true
+        };
+
+        const data = await this.fetchGraphQL(new URL('/graphql', this.url), operationName, query, variables);
+        return data.data.chapters.map((chapter) => ({
+            id: `${manga.id}/${chapter.number}`,
```

Comment:
It seems only the chapter number will be used in `_getPages` method, so for simplicity it should only use the chapter number as ID